public/app/features/dashboard/components/ShareModal/SharePublicDashboard/ConfigPublicDashboard/SettingsSummary.tsx

```diff
@@ -0,0 +1,57 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2, TimeRange } from '@grafana/data';
+import { Spinner, TimeRangeLabel, useStyles2 } from '@grafana/ui';
+
+export interface Props {
+  timeRange: TimeRange;
+  className?: string;
+  isDataLoading?: boolean;
+  timeSelectionEnabled?: boolean;
+  annotationsEnabled?: boolean;
+}
+
+export function SettingsSummary({
+  className,
+  isDataLoading = false,
+  timeRange,
+  timeSelectionEnabled,
+  annotationsEnabled,
+}: Props) {
+  const styles = useStyles2(getStyles);
+
+  return isDataLoading ? (
+    <div className={(styles.summaryWrapper, className)}>
```

Comment:
you would need cx to merge this classes

Reply:
oh yes, I don't know why was that even accepted